korman/exporter/mesh.py

```diff
@@ -94,6 +94,8 @@ def _determine_level(self, bo : bpy.types.Object, blend_span : bool) -> int:
             return self._calc_level(self.MAJOR_FRAMEBUF)
         elif mods.test_property("draw_opaque"):
             return self._calc_level(self.MAJOR_OPAQUE)
+        elif mods.test_property("draw_late"):
+            return self._calc_level(self.MAJOR_LATE)
```

Comment:
This also needs to be applied to the blending modifier's dependencies at line 110.

Reply:
Im trying to understand the logic here, wont it return before it can ever get there?

Reply:
If you have an object with the blending modifier that doesn't have a blend pass set but with render dependencies set, it may fall thorough to the case of checking the render dependencies below.